tests/virt/node/migration_and_maintenance/test_odf_vm_migration.py

```diff
@@ -23,6 +23,18 @@ def vm_with_common_cpu_model_scope_function(
         yield vm_from_template
 
 
+@pytest.fixture(scope="session")
+def xfail_test_if_no_odf_cephfs_sc(cluster_storage_classes_names):
```

Comment:
```suggestion
def xfail_if_no_odf_cephfs_sc(cluster_storage_classes_names):
```
here the fail in at setup level